keras_cv/models/__internal__/unet.py

```diff
@@ -177,6 +177,12 @@ def UNet(
     All function parameters require curried functions as inputs which return a
     function that acts on tensors as inputs.
 
+    Reference:
+        - [U-Net: Convolutional Networks for Biomedical Image Segmentation](https://arxiv.org/abs/1505.04597)
+
```

Comment:
We ended up with a lot of newlines here, let's just have one newline between the reference link and the `Args` section. Then I can merge 👍

Reply:
Removed The Extra Lines Between The Research Paper And The Code